src/index.js

```diff
@@ -0,0 +1,114 @@
+/* eslint no-use-before-define: */
+
+import './style.css';
+import List from './task.js';
+
+window.onload = () => {
```

Comment:
<img width="1010" alt="image" src="https://github.com/obibaadoma/To-Do-List/assets/27709832/98b2b234-1234-4ac6-a640-31377b61b1db">

As shown in this milestone description attached above, you are required to:
- Have an array of Objects with index(number), description(text), and completed(boolean) as key-values pairs.
- Loop through the array and render each object on the page, showing its description.
- Ensure they are rendered in the order of their indexes.

Kindly address each of these points.